VideoLocker/src/main/java/org/edx/mobile/module/serverapi/parser/GsonParser.java

```diff
@@ -0,0 +1,34 @@
+package org.edx.mobile.module.serverapi.parser;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+
+import java.io.Reader;
+import java.lang.reflect.Type;
+import java.util.List;
+
+/**
+ * Created by rohan on 2/6/15.
+ */
+class GsonParser implements IParser {
```

Comment:
Not at this moment, but we might want to do that in future. For example, <code>Jackson</code> is also a popular JSON parser, as compared to <code>Gson</code>.